pom.xml

```diff
@@ -35,10 +35,10 @@
 
         <version.grizzly-http-client>1.15</version.grizzly-http-client>
         <version.opentracing-api>0.31.0</version.opentracing-api>
-        <version.specialagent-maven-plugin>0.0.1-SNAPSHOT</version.specialagent-maven-plugin>
+        <version.specialagent>1.0.1-SNAPSHOT</version.specialagent>
```

Comment:
Shouldn't this be `1.0.1`?

Reply:
Apologies, this should actually be `1.0.2-SNAPSHOT`.

Reply:
This should now be `1.0.2`.